md/02.QuickStart/Ollama_QuickStart.md

```diff
@@ -26,7 +26,7 @@ If you want to call the Phi-3 API generated by ollama, you can use this command
 ollama serve
 
 ```
-***Note:***  If running MacOS or Linux, please note that you may encounter the following error <b>"Error: listen tcp 127.0.0.1:11434: bind: address already in use"</b> You may get this error when calling running the command. The solution for this problems is:
+***Note:***  If running MacOS or Linux, please note that you may encounter the following error <b>"Error: listen tcp 127.0.0.1:11434: bind: address already in use"</b> You may get this error when calling running the command. You can either ignore that error, since it typically indicates the server is already running, or you can stop the and restart Ollama:
```

Comment:
I never use this command, as the ollama server is always running in my env, so I reworded this note.